prototypes/ScheduledMerges.hs

```diff
@@ -375,12 +389,22 @@ combine x y = case x of
     Delete       -> Insert v Nothing
     Mupsert v'   -> Mupsert (resolveValue v v')
 
-lastLevelMerge :: Run -> Run
-lastLevelMerge = Map.filter (not . isDelete)
+-- | New value, old value.
+--
+-- TODO: implement directly on Ops?
```

Comment:
my guess is it'd be the same number of lines of code, but perhaps easier to grok